src/Makefile.am

```diff
@@ -153,7 +153,7 @@ geany_private.res: $(top_srcdir)/geany_private.rc
 
 libgeany_la_SOURCES += win32.c win32.h
 libgeany_la_LIBADD  += -lole32 -lwsock32 -lcomdlg32
-libgeany_la_LDFLAGS += -Wl,-luuid -mwindows -mms-bitfields -no-undefined
```

Comment:
`-luuid` is added by glib-2.0 on Mingw, so we don't need it here (maybe not anymore, this flag was added many years ago).